==== triton/BUILD#42 - /google/src/cloud/csigg/mlir_83e5a1239242d64110e3dfa96ed3889170ab96b2_1711020969/triton/BUILD ====
# action=edit type=text
--- triton/BUILD	2024-03-21 04:26:30.000000000 -0700
+++ triton/BUILD	2024-03-21 05:39:41.000000000 -0700
@@ -692,6 +692,7 @@
         "@llvm-project//mlir:SCFDialect",
         "@llvm-project//mlir:SCFTransforms",
         "@llvm-project//mlir:SCFUtils",
+        "@llvm-project//mlir:SideEffectInterfaces",
         "@llvm-project//mlir:Support",
         "@llvm-project//mlir:TensorDialect",
         "@llvm-project//mlir:TransformUtils",
